tools/wheels/cibw_before_build.sh

```diff
@@ -34,9 +41,18 @@ if [[ $RUNNER_OS == "macOS" ]]; then
         echo sha256 mismatch
         exit 1
     fi
+    
+    curl https://raw.githubusercontent.com/MacPython/gfortran-install/master/gfortran_utils.sh -o gfortran_utils.sh
```

Comment:
I would prefer to vendor this file. There's no license on this repo though https://github.com/MacPython/gfortran-install so not sure if I'm allowed to just copy it.

Reply:
@matthew-brett Looks like gfortran-install needs a license.